proxyprefix/wsgi.py

```diff
@@ -0,0 +1,13 @@
+class ReverseProxiedApp(object):
+    """WSGI middleware to prefix script name with X-Forwarded-Prefix header."""
+
+    def __init__(self, app):
+        self.app = app
```

Comment:
I thought since it's passed in, it's reasonable to expect it to be a public attribute. But I don't feel super strongly about it either way. If you do, let me know and I'll change it.